File: app/src/main/java/com/fj/temanteman/database/DatabaseContract.java
package com.fj.temanteman.database;

import android.provider.BaseColumns;

public class DatabaseContract {

    static String TABLE_FRIEND = "MsTeman";
    public static final class FriendColumns implements BaseColumns {
        public static String INITIAL = "initial";
        public static String NAME = "name";
        public static String MAJOR = "major";
        public static String PHONE = "phone";
    }
}

File: app/src/main/java/com/fj/temanteman/database/DatabaseContract.kt
package com.fj.temanteman.database

import android.provider.BaseColumns

object DatabaseContract {

    internal var TABLE_FRIEND = "MsTeman"

    class FriendColumns : BaseColumns {
        companion object {
            const val ID = "id"
            const val INITIAL = "initial"
            const val NAME = "name"
            const val MAJOR = "major"
            const val PHONE = "phone"
        }
    }
}
